genai/controlled_generation/controlled_generation_examples_test.go

```diff
@@ -25,7 +25,8 @@ func TestTextGeneration(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
 	t.Setenv("GOOGLE_GENAI_USE_VERTEXAI", "1")
-	t.Setenv("GOOGLE_CLOUD_LOCATION", "us-central1")
+	// t.Setenv("GOOGLE_CLOUD_LOCATION", "us-central1")
```

Comment:
Why are these just commented out?

Reply:
With in the `test` files, these are commented out as to act as a reference to know what is changes also to server back-up for future changes.